### ISSUE TITLE
Scaling relative to corner

### ISSUE THREAD

**Issue by adriankwiat** [publiclab/Leaflet.DistortableImage#403]
**Please describe the desired behavior.**
Describe the solution you'd like.

Image should be scaling relative to corner, not center, just like in Google Earth.

**Additional context (optional)**
Feel free to include any applicable: code, screenshots, or gifs.

In Google Earth:
![3](https://user-images.githubusercontent.com/36125672/63529306-c197c180-c504-11e9-9f00-dfc485cb948c.gif)

**Comment by sashadev-sky** [comment]
Oh I like this! This will be fun to make @rexagod

**Comment by jywarren** [comment]
Cool! This would be a constructor option on the tool itself, i guess? Maybe
"centroid" or "origin"?

**Comment by rexagod** [comment]
Sure @sashadev-sky! @jywarren I guess we might want to re-consider if we really need two different origin options? Can we think of a few test cases where one would be better than the other?

**Comment by sashadev-sky** [comment]
@jywarren @rexagod I don't think it's a need but I do think it improves usability!

**Comment by sashadev-sky** [comment]
**Uploaded to GCI dashboard**

**Comment by SidharthBansal** [comment]
Published

**Comment by Ridaoui02** [comment]
is there any hope we can get this feature plz!! ???

**Comment by joelthedangerous** [comment]
Yes i would love this feature too and it would save so much time for people that uses this tool to place an aerial overlay on a map. Its hard to adjust the overlay when it scales from the center when you drag a corner.
